lib/cartodb/backends/dataview.js

```
@@ -104,6 +114,12 @@ function getOverrideParams(params, ownFilter) {
         },
         {ownFilter: ownFilter}
     );
+
+    if (params.aggregation && DATE_AGGREGATIONS.hasOwnProperty(params.aggregation)) {
+        overrideParams.aggregation = params.aggregation;
```

We should throw an error definitely.